apps/portals/src/configurations/elportal/synapseConfigs/handleFilesToParticipants.tsx

```diff
@@ -9,10 +9,7 @@ import {
   CustomControlCallbackData,
 } from 'synapse-react-client'
 
-export const handleFilesToParticipants = async (
-  event: CustomControlCallbackData,
-) => {
-  // add filter for files perspective, to show participants associated to the selected files only.
+const getAllIDs = async (event: CustomControlCallbackData) => {
```

Comment:
Cohort Builder consists of 2 views - participant centric or files centric.
When no rows are selected, this queries for a single field (all pages) and passes the appropriate IDs to the other view.